src/lib/modules/ens/index.js

```diff
@@ -135,33 +135,38 @@ class ENS {
     this.embark.registerActionForEvent("deploy:beforeAll", this.configureContractsAndRegister.bind(this));
     this.events.on('blockchain:reseted', this.reset.bind(this));
     this.events.setCommandHandler("storage:ens:associate", this.associateStorageToEns.bind(this));
+    this.events.setCommandHandler("ens:config", this.getEnsConfig.bind(this));
+  }
+
+  getEnsConfig(cb) {
```

Comment:
I think this could be a simple return. Do you think the callback would be actually needed at some point?